packages/material-ui/src/Badge/Badge.test.js

```diff
@@ -170,4 +170,25 @@ describe('<Badge />', () => {
       expect(findBadge(container)).to.have.text('50');
     });
   });
+
+  it('retains anchorOrigin, content, color, max, overlap and variant when invisible is true for consistent disappearing transition', () => {
+    const wrapper = render(<Badge {...defaultProps} color="secondary" variant="dot" />);
+
+    act(() => {
```

Comment:
I was under the assumption that `setProps` already contains `act` (making it redundant), is this accurate?

Reply:
Here, found it: https://github.com/testing-library/react-testing-library/blob/6147830b0ea61341d02f520d2a26b817996e3a4a/src/pure.js#L60

Reply:
Good, will update the tests, wasn't aware of this